docs/workflow_tasks/move_granules.md

```diff
@@ -43,17 +41,13 @@ This task does expect values to be set in the CumulusConfig for the workflows.
 
 For the most recent config.json schema, please see the [Cumulus Tasks page](../tasks) entry for the schema.
 
-Below are expanded descriptions of selected config keys:
-
-#### Input_Granules
-
-An array of Cumulus [granule](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) objects.
+### Input
 
-This object will be used to define metadata values for the move granules task, and is the basis for the updated object that will be added to the output.
+This task expects event.input to provide an array of Cumulus [granule](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) objects.   The files listed for each granule represent the files to be acted upon as described in [summary](#summary).
 
 ## Task Outputs
 
-This task outputs an assembled array of Cumulus [granule](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) objects as the payload for the next task, and returns only the expected payload for the next task.
+This task outputs an assembled array of Cumulus [granule](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) objects as the payload for the next task, and returns only the expected payload for the next task.    If a CMR file has been specified for a granule, it will be updated with the appropriate values for it's relevant metadata.
```

Comment:
`granule objects with post-move file locations` or `with up-to-date filepaths`? might be nice to be clear about how the output is different from the input.

Reply:
Hmm, the issue here is that it's also updating the file to put the various 'online resources' (this isn't a good name, but matches the code....anyway) into the right types in the GMR file.  

How about 3e5e9da0a